hazenlib/spatial_resolution.py

```diff
@@ -424,6 +425,10 @@ def calculate_mtf_for_edge(dicom, edge, report_path=False):
     x, y = get_edge_profile_coords(angle, intercept, spacing)
     u, esf = get_esf(edge_arr, y)
     lsf = maivis_deriv(u, esf)
+    lsf = np.array(lsf)
+    n=lsf.size
+    freqs= fftfreq(n)
```

Comment:
This gives `freqs` in the units [1/pixels]. Try `freqs= fftfreq(n, d=profile_length/n)` to get in units [1/mm], which should fix the graph axis (line 469). You'll have to put this after the definition of profile_length in line 435.

This seems to give better units for the 256 and 512 images, but I could be wrong and welcome your thoughts.
256:
![image](https://user-images.githubusercontent.com/87647715/137479465-6b7b88c6-7ef6-46e7-89f9-f938ab7826ac.png)

512:
![image](https://user-images.githubusercontent.com/87647715/137479520-5229b718-2015-4e47-91f0-0d9b6148aa25.png)